src/ucp/rndv/proto_rndv.h

```diff
@@ -93,6 +93,18 @@ ucs_status_t
 ucp_proto_rndv_ctrl_init(const ucp_proto_rndv_ctrl_init_params_t *params);
 
 
+ucs_status_t
+ucp_proto_rndv_ctrl_init_common(const ucp_proto_rndv_ctrl_init_params_t *params,
```

Comment:
Please add brief function description, showing difference between using `ucp_proto_rndv_ctrl_init` and `ucp_proto_rndv_ctrl_init_common`, also to explain meaning of `common` postfix.